src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketState.java

```diff
@@ -94,6 +94,7 @@ public void addNotifierToBlockEvent(P context, IMixinWorldServer mixinWorldServe
     public void associateNeighborStateNotifier(P unwindingContext, BlockPos sourcePos, Block block, BlockPos notifyPos, WorldServer minecraftWorld,
         PlayerTracker.Type notifier) {
         final Player player = unwindingContext.getSpongePlayer();
+        unwindingContext.getOwner().ifPresent(owner -> ((IMixinChunk) minecraftWorld.getChunkFromBlockCoords(notifyPos)).setBlockCreator(notifyPos, owner.getUniqueId()));
```

Comment:
Same here.... use the player......... There's no other owner for a player, is there...